go.mod

```diff
@@ -13,4 +13,4 @@ require (
 	gopkg.in/yaml.v2 v2.4.0
 )
 
-go 1.15
+go 1.16
```

Comment:
We never agreed to move to Go 1.16.  This is not required for ZMQ

Reply:
my bad forgot to change back, updated!